engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-associations-content-hosts.html

```diff
@@ -28,9 +28,10 @@
         <tr bst-table-row ng-repeat="host in table.rows"
             ng-controller="ContentHostStatusController">
           <td bst-table-cell>
-            <a ui-sref="content-host.info({hostId: host.id})">
-              {{ host.name }}
-            </a>
+            <span ng-switch="newHostDetailsUI">
+              <a ng-switch-when="true" ng-href="/new/hosts/{{host.name}}">{{ host.name }}</a>
```

Comment:
Should we make all the links go to the Content tab like in Lucy's PR?

Reply:
I did activation keys/host collections to the main page since you can modify those there. I changed Errata to the Content - Errata, since it makes sense to see the Errata